README.md

```diff
@@ -1,10 +1,44 @@
 # kamon-graphite
 
 Write report metrics in graphite format.
-Per default the graphite tag support (available since graphite 1.1) is enabled (see include-tags feature flag).
+   
+## Usage
+Since kamon 2.0 you only have to include the dependency to the reporter.
+
+    libraryDependencies += "io.kamon" %% "kamon-graphite" % "2.0.0"
+
+Example config
+
+application.conf
+
+    kamon {
+      environment {
+        service = "supercool-app"
+        tags {
+          env = "local"
+        }
+      }
+      graphite {
+        hostname = "127.0.0.1"
+        port = 2003
+      }
+    }      
+
 
 ## Config
-    kamon{
+Per default the graphite tag support (available since graphite 1.1) is enabled (see include-tags feature flag).
+This is the default config which can be adapted to your needs using a custom `application.conf`.
+
+    kamon {
```

Comment:
This section of the configure probably shouldn't be here since people shouldn't really mess with this. Could you please either remove the `modules` section or reduce it to only mention the `enabled` setting which is the only one users could control.

Reply:
Sure! Reduced to only include the enabled flag.